Source/FLHook/Hook.h

```diff
@@ -160,9 +160,13 @@ struct PLUGIN_SORTCRIT {
 			if(itplugin->pFunc) { \
 				CTimer timer(itplugin->sPluginFunction,set_iTimerThreshold); \
 				timer.start(); \
+				auto timeHookStart = std::chrono::high_resolution_clock::now();\
 				try { \
 					vPluginRet = ((ret_type (calling_convention*) arg_types )itplugin->pFunc) args; \
 				} catch(...) { AddLog("ERROR: Exception in plugin '%s' in %s", itplugin->sName.c_str(), __FUNCTION__); LOG_EXCEPTION } \
+				if(set_hookPerfTimerLength && set_perfTimedHookName == __FUNCTION__) \
```

Comment:
do we not have a quicker way to check this than a string comparison?

Reply:
Afraid not, but note that it's prefaced with an UINT comparison which if failed (and it will fail unless logging is enabled), prevents the string comparison to begin with.